string command
---
bool success
string message
string output 